src/main/java/uk/gov/digital/ho/hocs/audit/service/AuditEventService.java

```diff
@@ -24,76 +27,57 @@ public class AuditEventService {
 
     private final AuditRepository auditRepository;
 
-    private final JsonValidator jsonValidator;
+    private final EntityManager entityManager;
 
     @Autowired
     public AuditEventService(AuditRepository auditRepository,
-                             JsonValidator jsonValidator) {
+                             EntityManager entityManager) {
         this.auditRepository = auditRepository;
-        this.jsonValidator = jsonValidator;
+        this.entityManager = entityManager;
     }
 
     public AuditEvent createAudit(String correlationID, String raisingService, String auditPayload, String namespace, LocalDateTime auditTimestamp, String type, String userID) {
         return createAudit(null, null, correlationID, raisingService, auditPayload, namespace, auditTimestamp, type, userID);
     }
 
     public AuditEvent createAudit(UUID caseUUID, UUID stageUUID, String correlationID, String raisingService, String auditPayload, String namespace, LocalDateTime auditTimestamp, String type, String userID) {
-        String validAuditPayload = jsonValidator.validateAuditPayload(correlationID, raisingService, auditPayload, namespace, auditTimestamp, type, userID);
```

Comment:
Not sure I agree with this, can this be moved to the constructor of AuditEvent?

If it's invalid do we get an exception thrown? If it ends up on the DLQ I suppose it's fine.

Reply:
IMO it's overly defensive and is another good example of a Developer checking getting into production, the database is 'not null' so everything will end up on the DLQ, and if it fails this check it ends up on the DLQ but now we have to keep these rules in line with the DB. It's not user input, so if it works in dev it'll work in prod and we don't need to check every message. if we HAVE to have it, it should be at the point the DTO is parsed.

Reply:
I'm guessing we are safe from sanitisation - while its not user input the data in the audit_payload can be. Do we sanitise?